packages/cpp/src/browser/cpp-language-client-contribution.ts

```diff
@@ -53,6 +56,9 @@ export class CppLanguageClientContribution extends BaseLanguageClientContributio
         // (e.g. compile_commands.json).
         this.onActiveBuildConfigChanged(this.cppBuildConfigurations.getActiveConfig());
         this.cppBuildConfigurations.onActiveConfigChange(config => this.onActiveBuildConfigChanged(config));
+
+        /** Display the C/C++ build configurations status bar element to select active build config */
```

Comment:
Comments in the code should use `// ...`.  jsdoc-style comments (to document classes, methods, etc) use `/** ... */`.